### PR TITLE
Change old username

### PR DESCRIPTION
**Pull request by alanrme**
fjah to alecks.

### CODE REVIEW

**Comment by XilogOfficial** [comment]
PLEASE COMMIT

File: site/src/install.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ case $(uname -s) in
 Darwin) target="macos" ;;
 *) target="linux" ;;
 esac
-
+https://github.com/alecks/pylon-cli/pull/1
```
Comment (alecks) [inline site/src/install.sh]: your clipboard works !

**Review by alecks — CHANGES_REQUESTED** [review]

File: site/src/install.sh (hunk 1 above)
Comment (alanrme) [reply site/src/install.sh]: Thanks for the kind words. I caused it to not work now

**Review by alecks — APPROVED** [review]

**Comment by alanrme** [comment]
THANKS U

**Pull request merged**